packages/replay-next/src/suspense/ScreenshotCache.ts

```diff
@@ -10,5 +10,18 @@ export const screenshotCache = createCache<
   config: { immutable: true },
   debugLabel: "ScreenshotCache",
   getKey: ([client, point, paintHash]) => paintHash,
-  load: ([client, point]) => client.getScreenshot(point),
+  load: async ([client, point]) => {
+    const screenShot = await client.getScreenshot(point);
+
+    paintHashCache.cacheValue(screenShot, screenShot.hash);
+
+    return screenShot;
```

Comment:
I'm not sure if this is prone to the same case mentioned in the repaint cache. Seems like it could be, depending on how RUN keys screenshots (by point or by hash)